utils/keys.go

```diff
@@ -93,7 +94,7 @@ func ExportKeys(to io.Writer, s Exporter, from string) error {
 // Each block is written to the subpath indicated in the "path" PEM
 // header. If the file already exists, the file is truncated. Multiple
 // adjacent PEMs with the same "path" header are appended together.
-func ImportKeys(from io.Reader, to []Importer) error {
+func ImportKeys(from io.Reader, to []Importer, role string, gun string, passRet notary.PassRetriever) error {
```

Comment:
nit:  can we call `role` and `gun` -> `fallbackRole` and `fallbackGun` or some such, to make it clear they're only used if there is no provided role or gun/path?